gui/default/assets/css/overrides.css

```diff
@@ -360,3 +360,8 @@ ul.three-columns li, ul.two-columns li {
 .fancytree-ext-table {
     width: 100% !important;
 }
+
+[id^="folder-"] th,
+[id^="device-"] th {
+    width: 200px;
+}
```

Comment:
Why is this needed?

Reply:
Even with the space gained by not showing some decimals, I still got truncated global states and at the same time there is unused white space on the left. With this, the white space is reduced, resulting in not truncated data.

Reply:
Where is the whitespace on the left coming from? Padding/margin?
Also, perhaps min-width then?

Reply:
The entire thing is a total can of worms, I was googling in dispair, because I didn't want to believe that it really is this bad: The table is fixed, because you can't set any size limits on auto, i.e. it exceeds its parent divs boundaries. min/max-width doesn't work on tables/td/th. Fixed with width set is the best solution I could find.

Reply:
I hope this doesn't screw up mobile.

Reply:
Possibly. I don't know and didn't test.

Reply:
If you own one, you could just navigate to the UI to your machine from your mobile.

Reply:
@AudriusButkevicius I removed this. After experimenting some more it became clear, that a good solution isn't simple and it's not directly related to the number mangling in this PR.